kratos/geometries/sphere_3d_1.h

```diff
@@ -402,7 +401,7 @@ class Sphere3D1 : public Geometry<TPointType>
     */
     double DomainSize() const override
     {
-        std::cout<<"This method (DomainSize) has no meaning for this type of geometry (Sphere)."<<std::endl;
+        KRATOS_WARNING("Sphere3D1") << "This method (DomainSize) has no meaning for this type of geometry (Sphere)." << std::endl;
```

Comment:
It was already here so I'd leave it as it is, but I think that in this case `DomainSize` does make sense.